pkg/services/dashboards/service/zanzana.go

```diff
@@ -195,3 +206,97 @@ func (dr *DashboardServiceImpl) checkDashboardsBatch(ctx context.Context, query
 
 	return result, nil
 }
+
+// findDashboardsZanzanaList implements "List, then search" strategy. It first retrieve a list of resources
+// with given type available to the user and then passes that list as a filter to the search query.
+func (dr *DashboardServiceImpl) findDashboardsZanzanaList(ctx context.Context, query dashboards.FindPersistedDashboardsQuery) ([]dashboards.DashboardSearchProjection, error) {
```

Comment:
I don't think this one behaves correctly. When query type is not set we need to search both dashboards and folders. This is not happening here

Reply:
Yes, this is true, I need to fix it.